TestMainUnknownCommand is flaky

```
travis_fold:start:worker_info
[0K[33;1mWorker information[0m
hostname: 94e8722f-ea17-4fc5-9dbb-45c29fa90840@1.worker-com-75b8f7d6f5-qxmfk.gce-production-1
version: v6.2.17 https://github.com/travis-ci/worker/tree/bc420764f334fe0d096a417c80426f672f1b7d18
instance: travis-job-497cbda0-5358-4edd-99ae-3e020c537e5a travis-ci-sardonyx-xenial-1593004276-4d46c6b3 (via amqp)
startup: 5.92047875s
travis_fold:end:worker_info
[0Ktravis_time:start:0aac1024
[0Ktravis_time:end:0aac1024:start=1595455071620492123,finish=1595455071787996116,duration=167503993,event=no_world_writable_dirs
[0Ktravis_time:start:1cf46019
[0Ktravis_time:end:1cf46019:start=1595455071791798796,finish=1595455071800018092,duration=8219296,event=agent
[0Ktravis_time:start:06e9eb02
[0Ktravis_time:end:06e9eb02:start=1595455071803382536,finish=1595455071806334743,duration=2952207,event=check_unsupported
[0Ktravis_time:start:0025273b
[0Ktravis_fold:start:system_info
[0K[33;1mBuild system information[0m
Build language: go
Build group: stable
Build dist: xenial
Build id: 176789829
Job id: 364030213
Runtime kernel version: 4.15.0-1077-gcp
travis-build version: 01e86bea
[34m[1mBuild image provisioning date and time[0m
Wed Jun 24 13:36:52 UTC 2020
[34m[1mOperating System Details[0m
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
[34m[1mSystemd Version[0m
systemd 229
[34m[1mCookbooks Version[0m
3f92a99 https://github.com/travis-ci/travis-cookbooks/tree/3f92a99
[34m[1mgit version[0m
git version 2.27.0
[34m[1mbash version[0m
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)
[34m[1mgcc version[0m
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
[34m[1mdocker version[0m
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:11:02 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.0-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       0ffa825
  Built:            Wed Jul 18 19:09:05 2018
  OS/Arch:          linux/amd64
  Experimental:     false
[34m[1mclang version[0m
clang version 7.0.0 (tags/RELEASE_700/final)
[34m[1mjq version[0m
jq-1.5
[34m[1mbats version[0m
Bats 0.4.0
[34m[1mshellcheck version[0m
0.7.0
[34m[1mshfmt version[0m
v2.6.3
[34m[1mccache version[0m
3.2.4
[34m[1mcmake version[0m
cmake version 3.12.4
[34m[1mheroku version[0m
heroku/7.42.1 linux-x64 node-v12.16.2
[34m[1mimagemagick version[0m
Version: ImageMagick 6.8.9-9 Q16 x86_64 2019-11-12 http://www.imagemagick.org
[34m[1mmd5deep version[0m
4.4
[34m[1mmercurial version[0m
version 4.8
[34m[1mmysql version[0m
mysql  Ver 14.14 Distrib 5.7.30, for Linux (x86_64) using  EditLine wrapper
[34m[1mopenssl version[0m
OpenSSL 1.0.2g  1 Mar 2016
[34m[1mpacker version[0m
1.3.3
[34m[1mpostgresql client version[0m
psql (PostgreSQL) 10.13 (Ubuntu 10.13-1.pgdg16.04+1)
[34m[1mragel version[0m
Ragel State Machine Compiler version 6.8 Feb 2013
[34m[1msudo version[0m
1.8.16
[34m[1mgzip version[0m
gzip 1.6
[34m[1mzip version[0m
Zip 3.0
[34m[1mvim version[0m
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Mar 18 2020 14:06:17)
[34m[1miptables version[0m
iptables v1.6.0
[34m[1mcurl version[0m
curl 7.47.0 (x86_64-pc-linux-gnu) libcurl/7.47.0 GnuTLS/3.4.10 zlib/1.2.8 libidn/1.32 librtmp/2.3
[34m[1mwget version[0m
GNU Wget 1.17.1 built on linux-gnu.
[34m[1mrsync version[0m
rsync  version 3.1.1  protocol version 31
[34m[1mgimme version[0m
v1.5.4
[34m[1mnvm version[0m
0.35.3
[34m[1mperlbrew version[0m
/home/travis/perl5/perlbrew/bin/perlbrew  - App::perlbrew/0.88
[34m[1mphpenv version[0m
rbenv 1.1.2-30-gc879cb0
[34m[1mrvm version[0m
rvm 1.29.10 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
[34m[1mdefault ruby version[0m
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
[34m[1mCouchDB version[0m
couchdb 1.6.1
[34m[1mElasticSearch version[0m
5.5.0
[34m[1mInstalled Firefox version[0m
firefox 63.0.1
[34m[1mMongoDB version[0m
MongoDB 4.0.19
[34m[1mPhantomJS version[0m
2.1.1
[34m[1mPre-installed PostgreSQL versions[0m
9.4.26
9.5.22
9.6.18
[34m[1mRedis version[0m
redis-server 6.0.5
[34m[1mPre-installed Go versions[0m
1.11.1
[34m[1mant version[0m
Apache Ant(TM) version 1.9.6 compiled on July 20 2018
[34m[1mmvn version[0m
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
[34m[1mgradle version[0m
Gradle 5.1.1!
[34m[1mlein version[0m
Leiningen 2.9.3 on Java 11.0.2 OpenJDK 64-Bit Server VM
[34m[1mPre-installed Node.js versions[0m
v10.21.0
v11.0.0
v12.18.1
v4.9.1
v6.17.1
v8.12.0
v8.17.0
v8.9
[34m[1mphpenv versions[0m
  system
  5.6
  5.6.40
  7.1
  7.1.27
  7.2
* 7.2.15 (set by /home/travis/.phpenv/version)
  hhvm
  hhvm-stable
[34m[1mcomposer --version[0m
Composer version 1.8.4 2019-02-11 10:52:10
[34m[1mPre-installed Ruby versions[0m
ruby-2.3.8
ruby-2.4.5
ruby-2.5.3
travis_fold:end:system_info
[0K
travis_time:end:0025273b:start=1595455071809727869,finish=1595455071816991719,duration=7263850,event=show_system_info
[0Ktravis_time:start:0ab6ceb1
[0Ktravis_time:end:0ab6ceb1:start=1595455071820439377,finish=1595455071835361766,duration=14922389,event=rm_riak_source
[0Ktravis_time:start:015624d1
[0Ktravis_time:end:015624d1:start=1595455071839201113,finish=1595455071845203704,duration=6002591,event=fix_rwky_redis
[0Ktravis_time:start:22ca1787
[0Ktravis_time:end:22ca1787:start=1595455071848982422,finish=1595455072228363802,duration=379381380,event=wait_for_network
[0Ktravis_time:start:35676ee2
[0Ktravis_time:end:35676ee2:start=1595455072231730656,finish=1595455072475521776,duration=243791120,event=update_apt_keys
[0Ktravis_time:start:209a70bc
[0Ktravis_time:end:209a70bc:start=1595455072479701942,finish=1595455072537070748,duration=57368806,event=fix_hhvm_source
[0Ktravis_time:start:31018614
[0Ktravis_time:end:31018614:start=1595455072540478641,finish=1595455072543407817,duration=2929176,event=update_mongo_arch
[0Ktravis_time:start:001babb0
[0Ktravis_time:end:001babb0:start=1595455072546642507,finish=1595455072595289011,duration=48646504,event=fix_sudo_enabled_trusty
[0Ktravis_time:start:291583aa
[0Ktravis_time:end:291583aa:start=1595455072599302375,finish=1595455072601998953,duration=2696578,event=update_glibc
[0Ktravis_time:start:0c0f955f
[0Ktravis_time:end:0c0f955f:start=1595455072605210603,finish=1595455072613765558,duration=8554955,event=clean_up_path
[0Ktravis_time:start:00321a6e
[0Ktravis_time:end:00321a6e:start=1595455072617092081,finish=1595455072626442408,duration=9350327,event=fix_resolv_conf
[0Ktravis_time:start:05546e04
[0Ktravis_time:end:05546e04:start=1595455072630203390,finish=1595455072640756309,duration=10552919,event=fix_etc_hosts
[0Ktravis_time:start:0c84e07a
[0Ktravis_time:end:0c84e07a:start=1595455072644266621,finish=1595455072655865670,duration=11599049,event=fix_mvn_settings_xml
[0Ktravis_time:start:0050983c
[0Ktravis_time:end:0050983c:start=1595455072659463658,finish=1595455072670047615,duration=10583957,event=no_ipv6_localhost
[0Ktravis_time:start:053cdf60
[0Ktravis_time:end:053cdf60:start=1595455072673499725,finish=1595455072676058775,duration=2559050,event=fix_etc_mavenrc
[0Ktravis_time:start:15b88b34
[0Ktravis_time:end:15b88b34:start=1595455072679570867,finish=1595455072682933902,duration=3363035,event=fix_wwdr_certificate
[0Ktravis_time:start:032ee9bc
[0Ktravis_time:end:032ee9bc:start=1595455072686105588,finish=1595455072712368466,duration=26262878,event=put_localhost_first
[0Ktravis_time:start:03462a98
[0Ktravis_time:end:03462a98:start=1595455072715634596,finish=1595455072718755683,duration=3121087,event=home_paths
[0Ktravis_time:start:170c8b92
[0Ktravis_time:end:170c8b92:start=1595455072722057304,finish=1595455072735232891,duration=13175587,event=disable_initramfs
[0Ktravis_time:start:00620e58
[0Ktravis_time:end:00620e58:start=1595455072738616404,finish=1595455073113111042,duration=374494638,event=disable_ssh_roaming
[0Ktravis_time:start:09503d80
[0Ktravis_time:end:09503d80:start=1595455073116569897,finish=1595455073119084013,duration=2514116,event=debug_tools
[0Ktravis_time:start:04f4dfb8
[0Ktravis_time:end:04f4dfb8:start=1595455073122831400,finish=1595455073126463202,duration=3631802,event=uninstall_oclint
[0Ktravis_time:start:0fb17348
[0Ktravis_time:end:0fb17348:start=1595455073129908848,finish=1595455073133337873,duration=3429025,event=rvm_use
[0Ktravis_time:start:25877a78
[0Ktravis_time:end:25877a78:start=1595455073136774732,finish=1595455073145699929,duration=8925197,event=rm_etc_boto_cfg
[0Ktravis_time:start:13d05f89
[0Ktravis_time:end:13d05f89:start=1595455073149247623,finish=1595455073152810800,duration=3563177,event=rm_oraclejdk8_symlink
[0Ktravis_time:start:05bf2825
[0Ktravis_time:end:05bf2825:start=1595455073156121761,finish=1595455073264198797,duration=108077036,event=enable_i386
[0Ktravis_time:start:1d1fab0c
[0Ktravis_time:end:1d1fab0c:start=1595455073267746879,finish=1595455073273815059,duration=6068180,event=update_rubygems
[0Ktravis_time:start:19937506
[0Ktravis_time:end:19937506:start=1595455073277354898,finish=1595455074124594565,duration=847239667,event=ensure_path_components
[0Ktravis_time:start:191cb1eb
[0Ktravis_time:end:191cb1eb:start=1595455074128068396,finish=1595455074130517010,duration=2448614,event=redefine_curl
[0Ktravis_time:start:0eb6d708
[0Ktravis_time:end:0eb6d708:start=1595455074133646879,finish=1595455074136134654,duration=2487775,event=nonblock_pipe
[0Ktravis_time:start:0684bfcd
[0Ktravis_time:end:0684bfcd:start=1595455074139353309,finish=1595455080176138135,duration=6036784826,event=apt_get_update
[0Ktravis_time:start:22e01c40
[0Ktravis_time:end:22e01c40:start=1595455080179617151,finish=1595455080182199072,duration=2581921,event=deprecate_xcode_64
[0Ktravis_time:start:16d6b580
[0Ktravis_time:end:16d6b580:start=1595455080185377194,finish=1595455082900234665,duration=2714857471,event=update_heroku
[0Ktravis_time:start:08ebf70e
[0Ktravis_time:end:08ebf70e:start=1595455082903847451,finish=1595455082906349187,duration=2501736,event=shell_session_update
[0Ktravis_time:start:1e9e135d
[0Ktravis_fold:start:docker_mtu
[0Ktravis_fold:end:docker_mtu
[0Ktravis_time:end:1e9e135d:start=1595455082909513814,finish=1595455085300216019,duration=2390702205,event=set_docker_mtu
[0Ktravis_time:start:148fc33e
[0Ktravis_fold:start:resolvconf
[0Ktravis_fold:end:resolvconf
[0Ktravis_time:end:148fc33e:start=1595455085306100780,finish=1595455085371327253,duration=65226473,event=resolvconf
[0Ktravis_time:start:09f20803
[0Ktravis_time:end:09f20803:start=1595455085378075218,finish=1595455085532634933,duration=154559715,event=maven_central_mirror
[0Ktravis_time:start:21553630
[0Ktravis_time:end:21553630:start=1595455085536395537,finish=1595455085641431512,duration=105035975,event=maven_https
[0Ktravis_time:start:00beb018
[0Ktravis_time:end:00beb018:start=1595455085645369320,finish=1595455085647787922,duration=2418602,event=fix_ps4
[0KUpdating gimme
travis_time:start:217f0e07
[0K
travis_fold:start:git.checkout
[0Ktravis_time:start:04c951b0
[0K$ git clone --depth=50 --branch=master https://github.com/GoogleContainerTools/skaffold.git GoogleContainerTools/skaffold
Cloning into 'GoogleContainerTools/skaffold'...
remote: Enumerating objects: 13386, done.[K
remote: Counting objects:   0% (1/13386)[K
remote: Counting objects:   1% (134/13386)[K
remote: Counting objects:   2% (268/13386)[K
remote: Counting objects:   3% (402/13386)[K
remote: Counting objects:   4% (536/13386)[K
remote: Counting objects:   5% (670/13386)[K
remote: Counting objects:   6% (804/13386)[K
remote: Counting objects:   7% (938/13386)[K
remote: Counting objects:   8% (1071/13386)[K
remote: Counting objects:   9% (1205/13386)[K
remote: Counting objects:  10% (1339/13386)[K
remote: Counting objects:  11% (1473/13386)[K
remote: Counting objects:  12% (1607/13386)[K
remote: Counting objects:  13% (1741/13386)[K
remote: Counting objects:  14% (1875/13386)[K
remote: Counting objects:  15% (2008/13386)[K
remote: Counting objects:  16% (2142/13386)[K
remote: Counting objects:  17% (2276/13386)[K
remote: Counting objects:  18% (2410/13386)[K
remote: Counting objects:  19% (2544/13386)[K
remote: Counting objects:  20% (2678/13386)[K
remote: Counting objects:  21% (2812/13386)[K
remote: Counting objects:  22% (2945/13386)[K
remote: Counting objects:  23% (3079/13386)[K
remote: Counting objects:  24% (3213/13386)[K
remote: Counting objects:  25% (3347/13386)[K
remote: Counting objects:  26% (3481/13386)[K
remote: Counting objects:  27% (3615/13386)[K
remote: Counting objects:  28% (3749/13386)[K
remote: Counting objects:  29% (3882/13386)[K
remote: Counting objects:  30% (4016/13386)[K
remote: Counting objects:  31% (4150/13386)[K
remote: Counting objects:  32% (4284/13386)[K
remote: Counting objects:  33% (4418/13386)[K
remote: Counting objects:  34% (4552/13386)[K
remote: Counting objects:  35% (4686/13386)[K
remote: Counting objects:  36% (4819/13386)[K
remote: Counting objects:  37% (4953/13386)[K
remote: Counting objects:  38% (5087/13386)[K
remote: Counting objects:  39% (5221/13386)[K
remote: Counting objects:  40% (5355/13386)[K
remote: Counting objects:  41% (5489/13386)[K
remote: Counting objects:  42% (5623/13386)[K
remote: Counting objects:  43% (5756/13386)[K
remote: Counting objects:  44% (5890/13386)[K
remote: Counting objects:  45% (6024/13386)[K
remote: Counting objects:  46% (6158/13386)[K
remote: Counting objects:  47% (6292/13386)[K
remote: Counting objects:  48% (6426/13386)[K
remote: Counting objects:  49% (6560/13386)[K
remote: Counting objects:  50% (6693/13386)[K
remote: Counting objects:  51% (6827/13386)[K
remote: Counting objects:  52% (6961/13386)[K
remote: Counting objects:  53% (7095/13386)[K
remote: Counting objects:  54% (7229/13386)[K
remote: Counting objects:  55% (7363/13386)[K
remote: Counting objects:  56% (7497/13386)[K
remote: Counting objects:  57% (7631/13386)[K
remote: Counting objects:  58% (7764/13386)[K
remote: Counting objects:  59% (7898/13386)[K
remote: Counting objects:  60% (8032/13386)[K
remote: Counting objects:  61% (8166/13386)[K
remote: Counting objects:  62% (8300/13386)[K
remote: Counting objects:  63% (8434/13386)[K
remote: Counting objects:  64% (8568/13386)[K
remote: Counting objects:  65% (8701/13386)[K
remote: Counting objects:  66% (8835/13386)[K
remote: Counting objects:  67% (8969/13386)[K
remote: Counting objects:  68% (9103/13386)[K
remote: Counting objects:  69% (9237/13386)[K
remote: Counting objects:  70% (9371/13386)[K
remote: Counting objects:  71% (9505/13386)[K
remote: Counting objects:  72% (9638/13386)[K
remote: Counting objects:  73% (9772/13386)[K
remote: Counting objects:  74% (9906/13386)[K
remote: Counting objects:  75% (10040/13386)[K
remote: Counting objects:  76% (10174/13386)[K
remote: Counting objects:  77% (10308/13386)[K
remote: Counting objects:  78% (10442/13386)[K
remote: Counting objects:  79% (10575/13386)[K
remote: Counting objects:  80% (10709/13386)[K
remote: Counting objects:  81% (10843/13386)[K
remote: Counting objects:  82% (10977/13386)[K
remote: Counting objects:  83% (11111/13386)[K
remote: Counting objects:  84% (11245/13386)[K
remote: Counting objects:  85% (11379/13386)[K
remote: Counting objects:  86% (11512/13386)[K
remote: Counting objects:  87% (11646/13386)[K
remote: Counting objects:  88% (11780/13386)[K
remote: Counting objects:  89% (11914/13386)[K
remote: Counting objects:  90% (12048/13386)[K
remote: Counting objects:  91% (12182/13386)[K
remote: Counting objects:  92% (12316/13386)[K
remote: Counting objects:  93% (12449/13386)[K
remote: Counting objects:  94% (12583/13386)[K
remote: Counting objects:  95% (12717/13386)[K
remote: Counting objects:  96% (12851/13386)[K
remote: Counting objects:  97% (12985/13386)[K
remote: Counting objects:  98% (13119/13386)[K
remote: Counting objects:  99% (13253/13386)[K
remote: Counting objects: 100% (13386/13386)[K
remote: Counting objects: 100% (13386/13386), done.[K
remote: Compressing objects:   0% (1/8069)[K
remote: Compressing objects:   1% (81/8069)[K
remote: Compressing objects:   2% (162/8069)[K
remote: Compressing objects:   3% (243/8069)[K
remote: Compressing objects:   4% (323/8069)[K
remote: Compressing objects:   5% (404/8069)[K
remote: Compressing objects:   6% (485/8069)[K
remote: Compressing objects:   7% (565/8069)[K
remote: Compressing objects:   8% (646/8069)[K
remote: Compressing objects:   9% (727/8069)[K
remote: Compressing objects:  10% (807/8069)[K
remote: Compressing objects:  11% (888/8069)[K
remote: Compressing objects:  12% (969/8069)[K
remote: Compressing objects:  13% (1049/8069)[K
remote: Compressing objects:  14% (1130/8069)[K
remote: Compressing objects:  15% (1211/8069)[K
remote: Compressing objects:  16% (1292/8069)[K
remote: Compressing objects:  17% (1372/8069)[K
remote: Compressing objects:  18% (1453/8069)[K
remote: Compressing objects:  19% (1534/8069)[K
remote: Compressing objects:  20% (1614/8069)[K
remote: Compressing objects:  21% (1695/8069)[K
remote: Compressing objects:  22% (1776/8069)[K
remote: Compressing objects:  23% (1856/8069)[K
remote: Compressing objects:  24% (1937/8069)[K
remote: Compressing objects:  25% (2018/8069)[K
remote: Compressing objects:  26% (2098/8069)[K
remote: Compressing objects:  27% (2179/8069)[K
remote: Compressing objects:  28% (2260/8069)[K
remote: Compressing objects:  29% (2341/8069)[K
remote: Compressing objects:  30% (2421/8069)[K
remote: Compressing objects:  31% (2502/8069)[K
remote: Compressing objects:  32% (2583/8069)[K
remote: Compressing objects:  33% (2663/8069)[K
remote: Compressing objects:  34% (2744/8069)[K
remote: Compressing objects:  35% (2825/8069)[K
remote: Compressing objects:  36% (2905/8069)[K
remote: Compressing objects:  37% (2986/8069)[K
remote: Compressing objects:  38% (3067/8069)[K
remote: Compressing objects:  39% (3147/8069)[K
remote: Compressing objects:  40% (3228/8069)[K
remote: Compressing objects:  41% (3309/8069)[K
remote: Compressing objects:  42% (3389/8069)[K
remote: Compressing objects:  43% (3470/8069)[K
remote: Compressing objects:  44% (3551/8069)[K
remote: Compressing objects:  45% (3632/8069)[K
remote: Compressing objects:  46% (3712/8069)[K
remote: Compressing objects:  47% (3793/8069)[K
remote: Compressing objects:  48% (3874/8069)[K
remote: Compressing objects:  49% (3954/8069)[K
remote: Compressing objects:  50% (4035/8069)[K
remote: Compressing objects:  51% (4116/8069)[K
remote: Compressing objects:  52% (4196/8069)[K
remote: Compressing objects:  53% (4277/8069)[K
remote: Compressing objects:  54% (4358/8069)[K
remote: Compressing objects:  55% (4438/8069)[K
remote: Compressing objects:  56% (4519/8069)[K
remote: Compressing objects:  57% (4600/8069)[K
remote: Compressing objects:  58% (4681/8069)[K
remote: Compressing objects:  59% (4761/8069)[K
remote: Compressing objects:  59% (4767/8069)[K
remote: Compressing objects:  60% (4842/8069)[K
remote: Compressing objects:  61% (4923/8069)[K
remote: Compressing objects:  62% (5003/8069)[K
remote: Compressing objects:  63% (5084/8069)[K
remote: Compressing objects:  64% (5165/8069)[K
remote: Compressing objects:  65% (5245/8069)[K
remote: Compressing objects:  66% (5326/8069)[K
remote: Compressing objects:  67% (5407/8069)[K
remote: Compressing objects:  68% (5487/8069)[K
remote: Compressing objects:  69% (5568/8069)[K
remote: Compressing objects:  70% (5649/8069)[K
remote: Compressing objects:  71% (5729/8069)[K
remote: Compressing objects:  72% (5810/8069)[K
remote: Compressing objects:  73% (5891/8069)[K
remote: Compressing objects:  74% (5972/8069)[K
remote: Compressing objects:  75% (6052/8069)[K
remote: Compressing objects:  76% (6133/8069)[K
remote: Compressing objects:  77% (6214/8069)[K
remote: Compressing objects:  78% (6294/8069)[K
remote: Compressing objects:  79% (6375/8069)[K
remote: Compressing objects:  80% (6456/8069)[K
remote: Compressing objects:  81% (6536/8069)[K
remote: Compressing objects:  82% (6617/8069)[K
remote: Compressing objects:  83% (6698/8069)[K
remote: Compressing objects:  84% (6778/8069)[K
remote: Compressing objects:  85% (6859/8069)[K
remote: Compressing objects:  86% (6940/8069)[K
remote: Compressing objects:  87% (7021/8069)[K
remote: Compressing objects:  88% (7101/8069)[K
remote: Compressing objects:  89% (7182/8069)[K
remote: Compressing objects:  90% (7263/8069)[K
remote: Compressing objects:  91% (7343/8069)[K
remote: Compressing objects:  92% (7424/8069)[K
remote: Compressing objects:  93% (7505/8069)[K
remote: Compressing objects:  94% (7585/8069)[K
remote: Compressing objects:  95% (7666/8069)[K
remote: Compressing objects:  96% (7747/8069)[K
remote: Compressing objects:  97% (7827/8069)[K
remote: Compressing objects:  98% (7908/8069)[K
remote: Compressing objects:  99% (7989/8069)[K
remote: Compressing objects: 100% (8069/8069)[K
remote: Compressing objects: 100% (8069/8069), done.[K
Receiving objects:   0% (1/13386)
Receiving objects:   1% (134/13386)
Receiving objects:   2% (268/13386)
Receiving objects:   3% (402/13386)
Receiving objects:   4% (536/13386)
Receiving objects:   5% (670/13386)
Receiving objects:   6% (804/13386)
Receiving objects:   7% (938/13386)
Receiving objects:   8% (1071/13386)
Receiving objects:   9% (1205/13386)
Receiving objects:  10% (1339/13386)
Receiving objects:  11% (1473/13386)
Receiving objects:  12% (1607/13386)
Receiving objects:  13% (1741/13386)
Receiving objects:  14% (1875/13386)
Receiving objects:  15% (2008/13386)
Receiving objects:  16% (2142/13386)
Receiving objects:  17% (2276/13386)
Receiving objects:  18% (2410/13386)
Receiving objects:  19% (2544/13386)
Receiving objects:  20% (2678/13386)
Receiving objects:  21% (2812/13386)
Receiving objects:  22% (2945/13386)
Receiving objects:  23% (3079/13386)
Receiving objects:  24% (3213/13386)
Receiving objects:  25% (3347/13386)
Receiving objects:  26% (3481/13386)
Receiving objects:  27% (3615/13386)
Receiving objects:  28% (3749/13386)
Receiving objects:  29% (3882/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  30% (4016/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  31% (4150/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  32% (4284/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  33% (4418/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  34% (4552/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  35% (4686/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  36% (4819/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  37% (4953/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  38% (5087/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  39% (5221/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  40% (5355/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  41% (5489/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  42% (5623/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  43% (5756/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  44% (5890/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  45% (6024/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  46% (6158/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  47% (6292/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  48% (6426/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  49% (6560/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  50% (6693/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  51% (6827/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  52% (6961/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  53% (7095/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  54% (7229/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  55% (7363/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  56% (7497/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  57% (7631/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  58% (7764/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  59% (7898/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  60% (8032/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  61% (8166/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  62% (8300/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  63% (8434/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  64% (8568/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  65% (8701/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  66% (8835/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  67% (8969/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  68% (9103/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  69% (9237/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  70% (9371/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  71% (9505/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  72% (9638/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  73% (9772/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  74% (9906/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  75% (10040/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  75% (10133/13386), 8.85 MiB | 17.69 MiB/s
Receiving objects:  76% (10174/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  77% (10308/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  78% (10442/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  79% (10575/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  80% (10709/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  81% (10843/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  82% (10977/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  83% (11111/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  84% (11245/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  85% (11379/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  86% (11512/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  87% (11646/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  88% (11780/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  89% (11914/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  90% (12048/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  91% (12182/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  92% (12316/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  93% (12449/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  94% (12583/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  95% (12717/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  96% (12851/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  97% (12985/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  98% (13119/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects:  99% (13253/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects: 100% (13386/13386), 22.05 MiB | 22.05 MiB/s
Receiving objects: 100% (13386/13386), 25.62 MiB | 21.41 MiB/s, done.
remote: Total 13386 (delta 6087), reused 9141 (delta 4516), pack-reused 0[K
Resolving deltas:   0% (0/6087)
Resolving deltas:   1% (120/6087)
Resolving deltas:   2% (168/6087)
Resolving deltas:   3% (228/6087)
Resolving deltas:   4% (279/6087)
Resolving deltas:   5% (309/6087)
Resolving deltas:   6% (367/6087)
Resolving deltas:   7% (437/6087)
Resolving deltas:   8% (487/6087)
Resolving deltas:   9% (568/6087)
Resolving deltas:  10% (609/6087)
Resolving deltas:  11% (673/6087)
Resolving deltas:  12% (737/6087)
Resolving deltas:  13% (801/6087)
Resolving deltas:  14% (853/6087)
Resolving deltas:  15% (959/6087)
Resolving deltas:  16% (982/6087)
Resolving deltas:  17% (1036/6087)
Resolving deltas:  18% (1103/6087)
Resolving deltas:  19% (1168/6087)
Resolving deltas:  20% (1220/6087)
Resolving deltas:  21% (1287/6087)
Resolving deltas:  22% (1345/6087)
Resolving deltas:  23% (1414/6087)
Resolving deltas:  24% (1465/6087)
Resolving deltas:  25% (1526/6087)
Resolving deltas:  28% (1745/6087)
Resolving deltas:  29% (1775/6087)
Resolving deltas:  30% (1829/6087)
Resolving deltas:  31% (1890/6087)
Resolving deltas:  32% (1950/6087)
Resolving deltas:  33% (2009/6087)
Resolving deltas:  34% (2071/6087)
Resolving deltas:  35% (2134/6087)
Resolving deltas:  36% (2192/6087)
Resolving deltas:  37% (2260/6087)
Resolving deltas:  38% (2318/6087)
Resolving deltas:  39% (2378/6087)
Resolving deltas:  40% (2438/6087)
Resolving deltas:  41% (2497/6087)
Resolving deltas:  42% (2571/6087)
Resolving deltas:  43% (2619/6087)
Resolving deltas:  44% (2679/6087)
Resolving deltas:  45% (2740/6087)
Resolving deltas:  46% (2801/6087)
Resolving deltas:  47% (2861/6087)
Resolving deltas:  48% (2922/6087)
Resolving deltas:  49% (2987/6087)
Resolving deltas:  50% (3046/6087)
Resolving deltas:  51% (3105/6087)
Resolving deltas:  52% (3166/6087)
Resolving deltas:  53% (3232/6087)
Resolving deltas:  54% (3287/6087)
Resolving deltas:  55% (3348/6087)
Resolving deltas:  56% (3428/6087)
Resolving deltas:  57% (3476/6087)
Resolving deltas:  58% (3532/6087)
Resolving deltas:  59% (3597/6087)
Resolving deltas:  60% (3656/6087)
Resolving deltas:  61% (3718/6087)
Resolving deltas:  62% (3778/6087)
Resolving deltas:  63% (3837/6087)
Resolving deltas:  64% (3897/6087)
Resolving deltas:  65% (3957/6087)
Resolving deltas:  66% (4020/6087)
Resolving deltas:  67% (4106/6087)
Resolving deltas:  68% (4152/6087)
Resolving deltas:  69% (4201/6087)
Resolving deltas:  70% (4264/6087)
Resolving deltas:  71% (4332/6087)
Resolving deltas:  72% (4383/6087)
Resolving deltas:  73% (4444/6087)
Resolving deltas:  74% (4511/6087)
Resolving deltas:  75% (4567/6087)
Resolving deltas:  76% (4627/6087)
Resolving deltas:  77% (4687/6087)
Resolving deltas:  78% (4748/6087)
Resolving deltas:  79% (4811/6087)
Resolving deltas:  80% (4870/6087)
Resolving deltas:  81% (4934/6087)
Resolving deltas:  82% (4993/6087)
Resolving deltas:  83% (5055/6087)
Resolving deltas:  84% (5114/6087)
Resolving deltas:  85% (5174/6087)
Resolving deltas:  86% (5235/6087)
Resolving deltas:  87% (5296/6087)
Resolving deltas:  88% (5357/6087)
Resolving deltas:  89% (5418/6087)
Resolving deltas:  90% (5499/6087)
Resolving deltas:  91% (5541/6087)
Resolving deltas:  92% (5653/6087)
Resolving deltas:  93% (5699/6087)
Resolving deltas:  94% (5731/6087)
Resolving deltas:  95% (5792/6087)
Resolving deltas:  96% (5844/6087)
Resolving deltas:  97% (5923/6087)
Resolving deltas:  98% (5966/6087)
Resolving deltas:  99% (6027/6087)
Resolving deltas: 100% (6087/6087)
Resolving deltas: 100% (6087/6087), done.
travis_time:end:04c951b0:start=1595455090153152878,finish=1595455094210860429,duration=4057707551,event=checkout
[0K$ cd GoogleContainerTools/skaffold
$ git checkout -qf a988b46c2104aaba8ef90c8a83b6a86b6eff0373
travis_fold:end:git.checkout
[0K
travis_time:end:04c951b0:start=1595455090153152878,finish=1595455094475789854,duration=4322636976,event=checkout
[0K$ travis_export_go 1.14.x /skaffold
travis_time:start:15d26c9c
[0Ktravis_time:end:15d26c9c:start=1595455094482656400,finish=1595455094493555449,duration=10899049,event=env
[0Ktravis_time:start:0472799e
[0K$ travis_setup_go
go version go1.14.6 linux/amd64
$ export GOPATH="/home/travis/gopath"
$ export PATH="/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.14.6.linux.amd64/bin:/home/travis/bin:/home/travis/bin:/home/travis/.local/bin:/usr/local/lib/jvm/openjdk11/bin:/opt/pyenv/shims:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v8.12.0/bin:/home/travis/.rvm/gems/ruby-2.5.3/bin:/home/travis/.rvm/gems/ruby-2.5.3@global/bin:/home/travis/.rvm/rubies/ruby-2.5.3/bin:/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.11.1.linux.amd64/bin:/usr/local/maven-3.6.3/bin:/usr/local/cmake-3.12.4/bin:/usr/local/clang-7.0.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/home/travis/.rvm/bin:/home/travis/.phpenv/bin:/opt/pyenv/bin:/home/travis/.yarn/bin"
$ export GO111MODULE="auto"
travis_time:end:0472799e:start=1595455094497460464,finish=1595455103549396938,duration=9051936474,event=
[0Ktravis_fold:start:cache.1
[0KSetting up build cache
$ export CASHER_DIR=${TRAVIS_HOME}/.casher
travis_time:start:0673e3e4
[0K$ Installing caching utilities
travis_time:end:0673e3e4:start=1595455104424733637,finish=1595455104534517864,duration=109784227,event=setup_casher
[0Ktravis_time:start:0e89ed26
[0Ktravis_time:end:0e89ed26:start=1595455104539967997,finish=1595455104543183789,duration=3215792,event=setup_casher
[0Ktravis_time:start:0a48c2c0
[0Kattempting to download cache archive[0m
[32;1mfetching master/cache--linux-xenial-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855--go-1.14.x.tgz[0m
[32;1mfound cache[0m
travis_time:end:0a48c2c0:start=1595455104547342194,finish=1595455107265442419,duration=2718100225,event=setup_casher
[0Ktravis_time:start:169c8ce8
[0Ktravis_time:end:169c8ce8:start=1595455107270361465,finish=1595455107273724189,duration=3362724,event=setup_casher
[0Ktravis_time:start:0316a359
[0Kadding /home/travis/.cache/go-build to cache[0m
travis_time:end:0316a359:start=1595455107278070745,finish=1595455145873262422,duration=38595191677,event=setup_casher
[0Ktravis_fold:end:cache.1
[0K
$ gimme version
v1.5.4
$ go version
go version go1.14.6 linux/amd64
travis_fold:start:go.env
[0K$ go env
GO111MODULE="auto"
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/travis/.cache/go-build"
GOENV="/home/travis/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/travis/gopath"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/home/travis/.gimme/versions/go1.14.6.linux.amd64"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/home/travis/.gimme/versions/go1.14.6.linux.amd64/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/travis/gopath/src/skaffold/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build740549907=/tmp/go-build -gno-record-gcc-switches"
travis_fold:end:go.env
[0Ktravis_fold:start:install
[0Ktravis_time:start:02ae7c93
[0K$ travis_install_go_dependencies 1.14.x -v
Makefile detected
travis_time:end:02ae7c93:start=1595455147100002774,finish=1595455147103476342,duration=3473568,event=install
[0Ktravis_fold:end:install
[0Ktravis_time:start:16144b57
[0K$ make coverage
mkdir -p ./out
go test -count=1 -race -cover -short -timeout=90s -coverprofile=out/coverage.txt -coverpkg=./pkg/...,./cmd/... ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/apiversion	0.161s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build	0.673s	coverage: 1.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/bazel	0.306s	coverage: 1.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/buildpacks	0.272s	coverage: 2.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache	0.847s	coverage: 3.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cluster	0.284s	coverage: 1.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/custom	0.175s	coverage: 3.1% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/gcb	0.284s	coverage: 2.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/jib	0.473s	coverage: 3.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/local	0.274s	coverage: 4.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/misc	0.871s	coverage: 0.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/tag	1.622s	coverage: 1.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/color	0.240s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/config	0.260s	coverage: 1.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/debug	1.078s	coverage: 5.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/kubectl	0.279s	coverage: 2.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy	1.336s	coverage: 13.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/resource	0.269s	coverage: 1.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/diagnose	1.284s	coverage: 4.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker	1.287s	coverage: 7.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/errors	0.280s	coverage: 0.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/event	2.814s	coverage: 2.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/filemon	0.278s	coverage: 1.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/generate_pipeline	0.243s	coverage: 1.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/gcp	0.248s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/initializer	0.455s	coverage: 4.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/initializer/analyze	0.318s	coverage: 2.1% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/initializer/build	0.305s	coverage: 1.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/initializer/deploy	0.272s	coverage: 1.0% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubectl	0.227s	coverage: 0.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes	0.334s	coverage: 1.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/debugging	0.192s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/context	0.300s	coverage: 0.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/generator	0.249s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/portforward	1.008s	coverage: 2.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/pipeline	0.307s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner	0.486s	coverage: 7.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext	0.291s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/util	0.237s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema	0.947s	coverage: 9.1% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/defaults	0.265s	coverage: 1.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util	0.232s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1	0.262s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha1	0.215s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha2	0.305s	coverage: 0.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha3	0.187s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha4	0.298s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha5	0.177s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta1	0.285s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta10	0.201s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta11	0.275s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta12	0.178s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta13	0.333s	coverage: 0.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta14	0.202s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta15	0.283s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta16	0.257s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta17	0.340s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta2	0.199s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta3	0.304s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta4	0.197s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta5	0.302s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta6	0.208s	coverage: 0.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta7	0.287s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta8	0.210s	coverage: 0.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta9	0.277s	coverage: 0.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha1	0.208s	coverage: 0.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha2	0.287s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha3	0.211s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha4	0.278s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2beta1	0.274s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2beta2	0.293s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2beta3	0.266s	coverage: 0.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2beta4	0.296s	coverage: 0.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/validation	0.244s	coverage: 1.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2beta5	0.298s	coverage: 0.5% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/survey	0.198s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/server	0.781s	coverage: 0.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/sync	0.231s	coverage: 3.1% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/test	0.273s	coverage: 1.7% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/test/structure	0.357s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/trigger	0.263s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/update	0.298s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/util	0.791s	coverage: 2.9% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/version	0.275s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/yamltags	0.267s	coverage: 0.8% of statements in ./pkg/..., ./cmd/...
=== CONT  
    testing.go:954: race detected during execution of test
[0;31mFAIL[0m	github.com/GoogleContainerTools/skaffold/cmd/skaffold/app	0.270s
ok  	github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd	1.190s	coverage: 11.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/config	0.331s	coverage: 1.6% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/schema	0.267s	coverage: 0.4% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags	0.257s	coverage: 0.8% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/hack/man	0.320s	coverage: 2.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/hack/versions/pkg/diff	0.470s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/hack/schemas	7.447s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/hack/versions/pkg/schema	0.274s	coverage: 0.2% of statements in ./pkg/..., ./cmd/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/webhook/labels	0.155s	coverage: 0.3% of statements in ./pkg/..., ./cmd/...
[0;31m
=== Failed Tests ===[0m
[1m/cmd/skaffold/app/TestMainUnknownCommand[0m
=== CONT  TestMainUnknownCommand
    testing.go:954: race detected during execution of test
--- [0;31mFAIL[0m: TestMainUnknownCommand (0.01s)
[1m/cmd/skaffold/app/TestMainUnknownCommand/TestMainUnknownCommand[0m
==================
WARNING: DATA RACE
Write at 0x0000041f0b24 by goroutine 35:
  github.com/spf13/pflag.newBoolValue()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/pflag/bool.go:16 +0x3e
  github.com/spf13/pflag.(*FlagSet).BoolVarP()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/pflag/bool.go:55 +0x4e
  github.com/spf13/pflag.(*FlagSet).BoolVar()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/pflag/bool.go:50 +0xd5d
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd.NewSkaffoldCommand()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/cmd/cmd.go:170 +0xcff
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app.Run()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/skaffold.go:32 +0x11e
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app.TestMainUnknownCommand.func1()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/skaffold_test.go:50 +0x16b
  github.com/GoogleContainerTools/skaffold/testutil.Run.func1()
      /home/travis/gopath/src/skaffold/testutil/util.go:219 +0x88
  testing.tRunner()
      /home/travis/.gimme/versions/go1.14.6.linux.amd64/src/testing/testing.go:1039 +0x1eb

Previous read at 0x0000041f0b24 by goroutine 27:
  github.com/GoogleContainerTools/skaffold/pkg/skaffold/update.IsUpdateCheckEnabled()
      /home/travis/gopath/src/skaffold/pkg/skaffold/update/update.go:51 +0xcb
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd.updateCheck()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/cmd/cmd.go:191 +0x78
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd.NewSkaffoldCommand.func2.1()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/cmd/cmd.go:97 +0x7c

Goroutine 35 (running) created at:
  testing.(*T).Run()
      /home/travis/.gimme/versions/go1.14.6.linux.amd64/src/testing/testing.go:1090 +0x700
  github.com/GoogleContainerTools/skaffold/testutil.Run()
      /home/travis/gopath/src/skaffold/testutil/util.go:218 +0xcd
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app.TestMainUnknownCommand()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/skaffold_test.go:47 +0x4c
  testing.tRunner()
      /home/travis/.gimme/versions/go1.14.6.linux.amd64/src/testing/testing.go:1039 +0x1eb

Goroutine 27 (running) created at:
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd.NewSkaffoldCommand.func2()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/cmd/cmd.go:96 +0x708
  github.com/spf13/cobra.(*Command).execute()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/cobra/command.go:821 +0xa52
  github.com/spf13/cobra.(*Command).ExecuteC()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/cobra/command.go:950 +0x499
  github.com/spf13/cobra.(*Command).Execute()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/cobra/command.go:887 +0x173
  github.com/spf13/cobra.(*Command).ExecuteContext()
      /home/travis/gopath/src/skaffold/vendor/github.com/spf13/cobra/command.go:880 +0x16a
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app.Run()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/skaffold.go:33 +0x17e
  github.com/GoogleContainerTools/skaffold/cmd/skaffold/app.TestMainHelp.func1()
      /home/travis/gopath/src/skaffold/cmd/skaffold/app/skaffold_test.go:37 +0x1cc
  github.com/GoogleContainerTools/skaffold/testutil.Run.func1()
      /home/travis/gopath/src/skaffold/testutil/util.go:219 +0x88
  testing.tRunner()
      /home/travis/.gimme/versions/go1.14.6.linux.amd64/src/testing/testing.go:1039 +0x1eb
==================
    testing.go:954: race detected during execution of test
    --- [0;31mFAIL[0m: TestMainUnknownCommand/TestMainUnknownCommand (0.00s)
[0;33m
=== Slow Tests ===[0m
7.13s	TestSchemas
2.14s	TestGetLogEvents
1.02s	TestCheckArtifacts/TestCheckArtifacts
1.02s	TestCheckArtifacts
0.63s	TestNewMinikubeImageAPIClient
0.61s	TestNewMinikubeImageAPIClient/correct_client
0.59s	TestNewCmdDebug
0.59s	TestNewCmdDebug/TestNewCmdDebug
0.54s	TestGetDeployments
0.51s	TestServerStartup
0.51s	TestGetDeployments/multiple_deployments_in_same_namespace
0.51s	TestCacheHasher/change_file_contents
0.51s	TestCacheHasher
0.51s	TestGracefulBuildCancel
0.40s	TestGracefulBuildCancel/kill_process_after_sigint
0.39s	TestApplyDebuggingTransforms
0.38s	TestCmd_RunCmdOut
0.38s	TestInParallelConcurrency
0.31s	TestParseExamples
0.29s	TestMonitorErrorLogs
exit status 1
Makefile:112: recipe for target 'coverage' failed
make: *** [coverage] Error 1
travis_time:end:16144b57:start=1595455147107848980,finish=1595455552978752278,duration=405870903298,event=script
[0K[31;1mThe command "make coverage" exited with 2.[0m
travis_fold:start:cache.2
[0Kstore build cache
travis_time:start:05fc0928
[0Ktravis_time:end:05fc0928:start=1595455552983570361,finish=1595455552986543063,duration=2972702,event=cache
[0Ktravis_time:start:0ac856fe
[0K[32;1mchanges detected (content changed, file is created, or file is deleted):\n/home/travis/.cache/go-build/00/004591bd04bd1916ccdf86907ac16364a8cf0691434ded1d05e8199c4e4e4f83-a
/home/travis/.cache/go-build/00/0045c4173023d27d8f9b53e5ef3ced306ad2de2bc1c0f223f24f79ca66f71496-a
/home/travis/.cache/go-build/00/0047b4c9239d8b2c832ed474787914876d87d30ed7218b5cdc3ac9b681e82ffa-a
/home/travis/.cache/go-build/00/00630fb56aea7e6e157804a22f3e1fa47068b85872b7d4ad86b9ef8c8eac6e6d-a
/home/travis/.cache/go-build/00/006e52f831bcdc720d16341b07a9574a1757952423d71d3bd0a35cefe1729639-a
/home/travis/.cache/go-build/00/0078739ba30ac0798239ca552594aee8c35a90358d880a58b3062e1904e9fc13-a
/home/travis/.cache/go-build/00/007dedc2094df6668def3722a9fecf0cc6098dafb0ff8a2c922f741cd88d2b4b-a
/home/travis/.cache/go-build/00/007f57fa40789196ab896f7a68eae589872a7bd8299cb8ebf2690524d7b71fec-a
/home/travis/.cache/go-build/00/0086bbf51b5ea79b9a6c8ab3909e3749085088bcd0469ad3c5d7687167141b73-a
/home/travis/.cache/go-build/00/0089fa0fd5ca42c744308bf3fbde060c0334da92641498210d129811ac8afdf6-a
/home/trav\n...[0m
[32;1mchanges detected, packing new archive[0m
[32;1muploading master/cache--linux-xenial-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855--go-1.14.x.tgz[0m
[32;1mcache uploaded[0m
travis_time:end:0ac856fe:start=1595455552990464905,finish=1595455591104377909,duration=38113913004,event=cache
[0Ktravis_fold:end:cache.2
[0K

Done. Your build exited with 1.
```

--- a/cmd/skaffold/app/skaffold_test.go
+++ b/cmd/skaffold/app/skaffold_test.go
@@ -27,13 +27,13 @@ import (
 
 func TestMainHelp(t *testing.T) {
 	testutil.Run(t, "", func(t *testutil.T) {
+		// --interactive=false removes the update check and survey prompt.
+		t.Override(&os.Args, []string{"skaffold", "help", "--interactive=false"})
+
 		var (
 			output    bytes.Buffer
 			errOutput bytes.Buffer
 		)
-
-		t.Override(&os.Args, []string{"skaffold", "help"})
-
 		err := Run(&output, &errOutput)
 
 		t.CheckNoError(err)
@@ -45,7 +45,8 @@ func TestMainHelp(t *testing.T) {
 
 func TestMainUnknownCommand(t *testing.T) {
 	testutil.Run(t, "", func(t *testutil.T) {
-		t.Override(&os.Args, []string{"skaffold", "unknown"})
+		// --interactive=false removes the update check and survey prompt.
+		t.Override(&os.Args, []string{"skaffold", "unknown", "--interactive=false"})
 
 		err := Run(ioutil.Discard, ioutil.Discard)
 

